src/components/dialogs/pick-request-type.vue

```diff
@@ -1,47 +1,34 @@
 <template>
   <v-dialog v-model="showModal"
             id="pick-request-type-modal"
-            max-width="600px"
+            max-width="45rem"
             hide-overlay>
-    <v-card class="pa-0" id="pick-request-type-modal-card">
-      <v-card-text class="h4 pa-3 px-5">What would you like to do?</v-card-text>
-      <v-container class="my-n4 pt-0">
-        <v-row>
-          <v-col cols="6">
-            <v-simple-table class="text-left">
-              <tr v-for="type in tableData.col1" :key="type.value+'-tr'">
-                <td class="clickable-cell" :id="type.value" @click="chooseType(type)">
-                  <v-tooltip bottom max-width="500" :disabled="isMobile">
-                    <template v-slot:activator="scope">
-                      <button class="link-sm-sans-ul" v-on="scope.on">{{ type.text }}</button>
-                    </template>
-                    <span>{{ type.blurb }}</span>
-                  </v-tooltip>
-                </td>
-              </tr>
-            </v-simple-table>
+    <v-card>
+      <v-card-title class="d-flex justify-space-between">
+        <h4>What would you like to do?</h4>
+        <v-btn icon large class="dialog-close" @click="showModal = false">
+          <v-icon>mdi-close</v-icon>
+        </v-btn>
+      </v-card-title>
+
+      <v-card-text>
+        <v-row no-gutters>
+          <v-col cols="6" class="d-block">
+            <ul v-for="(item, i) in tableData.col1" :key="`col1-${i}`">
```

Comment:
Question regarding the key here. If you don't have it, you get a "Elements in iteration expect to have 'v-bind:key' directives" error right?

Reply:
Something like that. All v-for items need a key to distinguish them and for reactivity to work (eg, list changes).

Reply:
OK, thanks. Resolving.